### ISSUE TITLE
ComboBox should have aria-invalid="true" on error

### ISSUE THREAD

**Issue by eivindland** [microsoft/fluentui#15958]
### Environment Information

- **Package version(s)**: Lastest 
- **Browser and OS versions**: Any

### Describe the issue:

When a ComboBox is in an error state - that is, has an errorMessage-prop set - a screen reader does not recognize the input field as being in an error state. 

### Please provide a reproduction of the issue in a codepen:

Visible at: https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

#### Actual behavior:

The input element in ComboBox does not have any _aria-invalid_ attribute to tell the screen reader about the error state. 

#### Expected behavior:

I would expect the input element to have an aria-invalid attribute that swtiches from "false" to "true" when it goes from the non-error-state to the error state. 
(Or at least a way to add this attribute:  aria-invalid={errorMessage ? true : false})

### Documentation describing expected behavior
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute

**Comment by paulgildea** [comment]
@eivindland Thanks for filling this issue we'll look into it!

@smhigley FYI
